src/sentry/static/sentry/app/components/notAvailable.tsx

```diff
@@ -1,8 +1,21 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
-function NotAvailable() {
-  return <Wrapper>{'\u2014'}</Wrapper>;
+import Tooltip from 'app/components/tooltip';
+import {defined} from 'app/utils';
+
+type Props = {
+  tooltip?: string;
```

Comment:
maybe we would like to pass a `tct` translation here and unfortunately, the return type is not `string`, so I think we should update it to be `React.ReactNode `